OnboardPietroPieri/utils/render.tsx

```diff
@@ -1,13 +1,21 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
 import { User } from "../interfaces/mutation";
 
-export const renderItem = ({ item }: { item: User }) => (
-  <View style={styles.userItem}>
-    <Text style={styles.userName}>{item.name}</Text>
+export const renderItem = ({
+  item,
+  onPress,
+}: {
+  item: User;
+  onPress: (id: string) => void;
+}) => (
+  <TouchableOpacity onPress={() => onPress(item.id)}>
```

Comment:
Nit: No lugar que você está usando o renderItem, está ignorando o id, então poderia remover daqui
```suggestion
  onPress: () => void;
}) => (
  <TouchableOpacity onPress={onPress}>
```